docs/notebooks/introduction.pct.py

```diff
@@ -131,6 +131,8 @@
 
 dataset = result.datasets[OBJECTIVE]
 
+ei_optimization_progress = np.minimum.accumulate(dataset.observations)[5:]
```

Comment:
would this be clearer further down, closer to where we use it? you may need to rename `dataset` further down if you do